openedx/core/djangolib/js_utils.py

```diff
@@ -9,7 +9,7 @@
 from xmodule.modulestore import EdxJSONEncoder
 
 
-def _escape_json_for_js(json_dumps_string):
+def escape_json_for_js(json_dumps_string):
```

Comment:
Just following the convention here. We don't need weak internal usage indicator (_ before method name) as we are importing this method directly now and there can be other use cases where we only need this method.
Any suggestions would be appreciated.